plugins/crm/webapp/crm/WEB-INF/controller.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+<site-conf xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xmlns="http://ofbiz.apache.org/Site-Conf" xsi:schemaLocation="http://ofbiz.apache.org/Site-Conf http://ofbiz.apache.org/dtds/site-conf.xsd">
+    <!-- The controller elements that are common to all OFBiz components
+         can be found in the following xml file. A component can override the
+         elements found in the common-controller.xml file. -->
+    <include location="component://common/webcommon/WEB-INF/common-controller.xml"/>
+
+    <description>CrmClientView Component Site Configuration File</description>
+
+    <!-- Events to run on every request before security (chains exempt) -->
+    <!--
+    <preprocessor>
+    </preprocessor>
+    -->
+    <!-- Events to run on every request after all other processing (chains exempt) -->
+    <!--
+    <postprocessor>
+        <event name="test" type="java" path="org.apache.ofbiz.webapp.event.TestEvent" invoke="test"/>
+    </postprocessor>
+    -->
+
+    <!-- Request Mappings -->
+    <request-map uri="main"><security https="true" auth="true"/><response name="success" type="view" value="main"/></request-map>
+
+    <request-map uri="FindContacts">
+        <security https="true" auth="true"/>
+        <response name="success" type="view" value="FindContacts" save-home-view="true"/>
+    </request-map>
+
+    <request-map uri="getContactsList">
+        <security https="true" auth="false"/>
+        <!--        <event type="service" invoke="getContactsListService"/>-->
+        <event type="java" path="com.taltech.crm.services.ContactsService" invoke="getContactList"/>
+        <!--        <response name="success" type="view" value="FindContacts"/>-->
+        <response name="success" type="request" value="json"/>
+        <response name="error" type="view" value="main"/>
+    </request-map>
+
+    <!--    ADDED-->
+    <request-map uri="createContact">
+        <security https="true" auth="false"/>
+        <event type="service" invoke="createContact"/>
+        <response name="success" type="request" value="json"/>
+    </request-map>
```

Comment:
Not sure if your using camel or vanilla ofbiz for json api. Mostlikely could remove one